core/integration/module_test.go

```diff
@@ -3395,6 +3395,70 @@ func (t *Toplevel) SayHello(ctx context.Context) (string, error) {
 	require.JSONEq(t, `{"toplevel":{"sayHello":"hello!"}}`, out)
 }
 
+func TestModuleGoCustomScalar(t *testing.T) {
```

Comment:
If you can, try to design tests that can be shared between the sdks, like some do. Already setting up the testcase loop and thinking of the test in an agnostic manner really helps. Scalar and enum support are especially well suited for that. If there's some sdk specific idiosyncracies, we can still have a common test and other sdk specific subtests or tests. There's several cases of this.

To be clear, I'm not saying these test steps aren't transferrable, I'm speaking in more general terms and only specifically about preparing the `[]testcase`. It helps to bring in the required SDK consistency on board. 😄

Reply:
This specific test is now removed since I've removed the custom scalars, but I've now added a table test for custom scalar args from core.